src/store/safeAppsSlice.ts

```diff
@@ -1,17 +1,20 @@
 import type { PayloadAction } from '@reduxjs/toolkit'
 import { createSelector } from '@reduxjs/toolkit'
 import { createSlice } from '@reduxjs/toolkit'
+import type { SafeAppData } from '@safe-global/safe-gateway-typescript-sdk'
 import type { RootState } from '@/store'
 
 type SafeAppsPerChain = {
-  pinned: number[]
+  pinned: Array<SafeAppData['id']>
 }
 
 export type SafeAppsState = {
   [chainId: string]: SafeAppsPerChain
-}
+} & { opened: Array<SafeAppData['id']> }
```

Comment:
Wait, you mixed it in along with chainIds? That's no bueno...

Reply:
As per discussion, I've reverted the previous commit as `chainId`s are top level.